bridge/src/lib.rs

```diff
@@ -155,6 +159,26 @@ methods!(
 
         NilClass::new()
     }
+
+    fn worker_complete_activity_task(proto: RString) -> NilClass {
+        if !VM::is_block_given() {
+            panic!("Called #worker_complete_activity_task without a block");
+        }
+
+        let bytes = unwrap_bytes(proto.map_err(VM::raise_ex).unwrap());
```

Comment:
Are we okay with all of these `unwrap` statements?
Are they "safe"?

Reply:
Yep, since we have `map_err` raising exception in Ruby in case of an `Err` value